docs/marina/api.md

```diff
@@ -128,9 +134,19 @@ marina.getBalances(): Promise<Balance[]>;
 ### on
 
 ```typescript
-marina.on(type: MarinaEventType, callback: (payload: any) => void): void;
+marina.on(type: MarinaEventType, callback: (payload: any) => void): EventListenerID;
+```
```

Comment:
worth documenting all the events of MarinaEventType in a dedicated section maybe